index.html

```diff
@@ -0,0 +1,228 @@
+                    </div>
+                    <p class="course-item-description">in this section of the
+                        program we talk
+                        about back-end development in general and in ruby on
+                        rails especially.
+                    </p>
+                </li>
+            </ul>
+            <a href="#">SEE THE WHOLE PROGRAM</a>
+            <button>Join Our 2021 Webinar</button>
+        </section>
+
+        <section id="featured-speakers">
+            <h2 class="text-center">Featuerd Speakers</h2>
+            <div id="hr-wrapper">
+                <hr>
+            </div>
+            <ul>
+                <li>
+                    <img src="images/speaker-1.png" alt="speaker number one">
+                    <div>
+                        <h3 class="bold speaker-name"></h3>
+                        <p class="orange-color bold speaker-job"><p>
+                                <hr class="speaker-hr">
+                                <p class="speaker-graduated"></p>
+                            </div>
+                        </li>
+                        <li>
+                            <img src="images/speaker-2.png" alt="speaker number
+                                one">
+                            <div>
+                                <h3 class="bold speaker-name"></h3>
+                                <p class="orange-color bold speaker-job"></p>
+                                <hr class="speaker-hr">
+                                <p class="speaker-graduated"></p>
+                            </div>
+                        </li>
+                        <li class="hide-on-mobile">
+                            <img src="images/speaker-3.png" alt="speaker number
+                                one">
+                            <div>
+                                <h3 class="bold speaker-name"></h3>
+                                <p class="orange-color bold speaker-job"></p>
+                                <hr class="speaker-hr">
+                                <p class="speaker-graduated"></p>
+                            </div>
+                        </li>
+                        <li class="hide-on-mobile">
+                            <img src="images/speaker-4.png" alt="speaker number
+                                one">
+                            <div>
+                                <h3 class="bold speaker-name"></h3>
+                                <p class="orange-color bold speaker-job"></p>
+                                <hr class="speaker-hr">
+                                <p class="speaker-graduated"></p>
+                            </div>
+                        </li>
+                    </ul>
+                    <div id="more-button-wrapper">
+                        <button>More <i class="bi bi-chevron-down orange-color"></i></button>
+                    </div>
+                </section>
```

Comment:
- [ ] Pictures of speakers of your _feature speakers_ section are missing the background decoration, as can be seen in the template design. Kindly ensure to improve upon this aspect by adding the background decorations as well as necessary adjustments to make it look like a replica of the template design.

**Your feature speakers section:**
![image](https://user-images.githubusercontent.com/12745474/132546876-9635ecce-70de-4dd0-ac58-fac1c4973f09.png)

**The original feature speakers section:**
![image](https://user-images.githubusercontent.com/12745474/132547059-14af2209-d5e7-48b5-9e41-622ed7e3ebea.png)